analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -363,14 +363,19 @@ abstract class PackageManager(
 }
 
 /**
- * Parse a string with metadata about an [author] to extract the author name. Many package managers support
- * such author information in string form that contain additional properties like an email address or a
- * homepage. These additional properties are typically separated from the author name by specific [delimiters],
- * e.g. the email address is often surrounded by angle brackets. This function assumes that the author name is the
- * first portion in the given [author] string before one of the given [delimiters] is found.
+ * Parse a string with metadata about an [author] that several package managers use and try to extract the author's
+ * name, email address, and homepage. These properties are typically surrounded by specific [delimiters], e.g. the email
+ * address is often surrounded by angle brackets. Return a triple for these properties where unavailable ones are set to
+ * empty strings.
  */
-fun parseAuthorString(author: String?, vararg delimiters: Char = charArrayOf('<')): String? =
-    author?.split(*delimiters, limit = 2)?.firstOrNull()?.trim()?.ifEmpty { null }
+fun parseAuthorString(
```

Comment:
Would you be ok to add a `data` class for the return type for readability / self explanatory?

Reply:
While I agree it would be a bit cleaner by the book, I didn't want to basically duplicate the `PackageJson.Author` class here and introduce a dedicated class just for use by this single function.

Reply:
I've changed my mind as I need to improve the parsing anyway. Will send a new iteration shortly.